RELEASE-NOTES.md

```diff
@@ -10,6 +10,7 @@ The Uberenv project release numbers follow [Semantic Versioning](http://semver.o
 ## Unreleased
 
 ### Added
+- Adds `--package_final_phase` option for install.
```

Comment:
Please (briefly) expand this to indicate what this new option does

Reply:
It's not technically a new option, it is adding a feature that the dev-build build type had to the install build type.